src/app/services/validation.ts

```diff
@@ -21,6 +21,22 @@ export const validateEmail = (email?: string) => {
     return email && email.length > 0 && email.includes("@");
 };
 
+ // function that returns true if email exists and is not on the nonSchoolDomains list
```

Comment:
was this list of domains in the react source code before?

Reply:
As far as I'm aware there isn't a check on the back end and this was already in the code. It's not really a very robust check